color/src/colorspace.rs

```diff
@@ -230,6 +267,53 @@ impl ColorSpace for DisplayP3 {
     }
 }
 
+/// The CIE XYZ color space with a 2° observer and a reference white of D65.
+///
+/// Its components are `[X, Y, Z]`. The components are unbounded, but are usually positive.
+/// Reference white has a luminance `Y` of 1.
+///
+/// This corresponds to the color space in [CSS Color Module Level 4 § 10.8][css-sec]. It is
+/// defined in CIE 015:2018.
+///
+/// # Human color vision and color spaces
+///
+/// Human color vision uses three types of photoreceptive cell in the eye that are sensitive to
+/// light. These cells have their peak sensitivity at different wavelengths of light: roughly 570
+/// nm, 535 nm and 430 nm, usually named Long, Medium and Short (LMS) respectively. The cells'
+/// sensitivities to light taper off as the wavelength moves away from their peaks, but all three
+/// cells overlap in wavelength sensitivity.
+///
+/// Visible light with a combination of wavelengths at specific intensities (the light's *spectral
+/// density*), causes excitation of these three cell types in varying amounts. The human brain
+/// interprets this as a specific color at a certain luminosity. Importantly, humans do not
+/// directly perceive the light's wavelength: for example, monochromatic light with a wavelength of
+/// 580 nm is perceived as "yellow," and light made up of two wavelengths at roughly 550nm
+/// ("green") and 610 nm ("red") is also perceived as "yellow."
+///
+/// The CIE XYZ color space is an experimentally-obtained mapping of monochromatic light at a
+/// specific wavelength to the response of human L, M and S photoreceptive cells (with some
+/// additional mathematically desirable properties). Light of a specific spectral density maps onto
+/// a specific coordinate in the XYZ color space. Light of a different spectral density that maps
+/// onto the same XYZ coordinate is predicted by the color space to be perceived as the same
+/// color and luminosity.
+///
+/// The XYZ color space is often used in the characterization of other color spaces.
+///
+/// ## White point
+///
+/// An important concept in color spaces is the *white point*. Whereas pure black is the absence of
+/// illumination and has a natural representation in additive color spaces, white is more difficult
+/// to define. CIE D65 defines white as the perceived color of diffuse standard noon daylight
+/// perfectly reflected off a surface observed under some foveal angle; here 2°.
+///
+/// In many color spaces, their white point is the brightest illumination they can naturally
+/// represent.
+///
+/// For further reading, the [Wikipedia article on the CIE XYZ color space][wikipedia-cie] provides
+/// a good introduction to color theory as relevant to color spaces.
```

Comment:
Is this correct/clear/necessary/sufficient?

Reply:
Yes, I think so, and I think this is the right place for such material. Thanks for writing it up!